package.json

```diff
@@ -7,28 +7,29 @@
 		"node": "^14.19 || ^16.14"
 	},
 	"scripts": {
-		"postinstall": "yarn install:packages && yarn install:meteor",
+		"postinstall": "cd packages && yarn postinstall && cd ../meteor && yarn postinstall",
+		"install": "yarn install:packages && yarn install:meteor",
 		"install:meteor": "cd meteor && meteor --version && meteor npm install -g yarn && node ../scripts/fix-windows-yarn.js && meteor yarn install",
 		"install:packages": "cd packages && yarn install",
 		"start": "yarn install && yarn dev",
 		"dev": "node ./scripts/run.js",
 		"watch": "yarn dev",
 		"restart:meteor": "node ./scripts/meteor-force-restart.js",
-		"build": "yarn install && yarn build:packages",
+		"build": "yarn build:packages",
 		"build:packages": "cd packages && yarn build",
 		"test:packages": "cd packages && yarn test",
 		"lint:packages": "cd packages && yarn lint",
-		"lint-fix:packages": "cd packages && yarn lint --fix",
+		"lint-fix:packages": "cd packages && yarn lint-fix",
 		"unit:packages": "cd packages && yarn unit",
 		"check-types:meteor": "cd meteor && yarn check-types",
 		"test:meteor": "cd meteor && yarn test",
 		"lint:meteor": "cd meteor && meteor yarn lint",
-		"lint-fix:meteor": "cd meteor && meteor yarn lint --fix",
+		"lint-fix:meteor": "cd meteor && yarn lint-fix",
```

Comment:
I'm not sure that this has the same behaviour anymore.
Meteor has its own yarn instance so running `yarn install` and `meteor yarn install` is using two different yarn installations and they install two different places.
This is not my area of expertise, so can't say for sure if this change breaks anything.

Reply:
great catch!